lazyflow/utility/pathHelpers.py

```diff
@@ -413,3 +423,9 @@ def globNpz(path: str, globString: str):
 def globList(listOfPaths, globString):
     matches = [x for x in listOfPaths if fnmatch.fnmatch(x, globString)]
     return matches
+
+
+if __name__ == "__main__":
+    import doctest
+
+    doctest.testmod()
```

Comment:
can be removed; we have removed those main sections in the tests (or well started to do so), so it is against that "style" to add those main functions here.

```bash
pytest --doctest-modules ...
```

Reply:
Moved tests out of doctest and into `tests/`